src/views/GRTSystemIntakeReview/index.tsx

```diff
@@ -17,30 +25,118 @@ export const GRTSystemIntakeReview = () => {
   const systemIntake = useSelector(
     (state: AppState) => state.systemIntake.systemIntake
   );
+  const isLoading = useSelector(
+    (state: AppState) => state.systemIntake.isLoading
+  );
   return (
     <div>
       <Header />
       <MainContent className="grid-container">
         <div className="system-intake__review margin-bottom-7">
           <h1 className="font-heading-xl margin-top-4">CMS System Request</h1>
+          {/* TODO: systemIntake is always truthy */}
```

Comment:
Can you say more about this TODO?

Reply:
Best I can understand, the intent of this line is "if the intake is not successfully fetched from the server, instead display this error" but in fact, the `systemIntake` object there is always truthy, so the condition `!systemIntake` will never be met: even if the fetch fails, the `initialData` of the redux store still has an empty systemIntake in it

Reply:
I've used systemIntakeId as a way to check if it's an official systemIntake.